npm/pkg/controlplane/controllers/v2/namespaceController.go

```diff
@@ -221,7 +221,7 @@ func (nsc *NamespaceController) processNextWorkItem() bool {
 }
 
 // syncNamespace compares the actual state with the desired, and attempts to converge the two.
-func (nsc *NamespaceController) syncNamespace(nsKey string) error {
+func (nsc *NamespaceController) syncNamespace(nsKey string) (err error) {
```

Comment:
Can we fix this golint.

Reply:
discussed offline that we shouldn't return sync errors when the dataplane fails. The dataplane should reconcile on its own